pkgs/by-name/l8/l8w8jwt/package.nix

```diff
@@ -0,0 +1,58 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "l8w8jwt";
+  version = "2.5.0";
+
+  src = fetchFromGitHub {
+    owner = "GlitchedPolygons";
+    repo = "l8w8jwt";
```

Comment:
Looks like this upstream was migrated to codeberg recently:

https://github.com/GlitchedPolygons/l8w8jwt?tab=readme-ov-file

> Development of this project has been migrated to: https://codeberg.org/GlitchedPolygons/l8w8jwt

Reply:
Fixed in the latest rebase + push.